client/my-sites/plugins/plugins-browser-list/test/index.jsx

```diff
@@ -25,29 +20,38 @@ const plugins = [
 const props = {
 	plugins,
 	variant: PluginsBrowserListVariant.Fixed,
-	listName: 'woocommerce',
-	title: 'woocommerce',
-	subtitle: '100 plugins',
+	listName: 'arbitrary-plugin-list-name',
+	title: 'arbitrary-plugin-list-title',
+	subtitle: 'arbitrary-plugin-list-subtitle',
 	site: {
 		plan: PLAN_FREE,
 	},
 	size: 6,
 };
 
+const queryPlaceholders = () => {
+	const listItems = screen.queryAllByRole( 'listitem' );
+	return listItems.filter( ( item ) => Array.from( item.classList ).includes( 'is-placeholder' ) );
+};
+
 describe( 'PluginsBrowserList basic tests', () => {
 	test( 'should render the section header with title', () => {
-		const comp = shallow( <PluginsBrowserList { ...props } /> );
-		expect( comp.find( '.plugins-browser-list__title' ).text() ).toBe( 'woocommerce' );
+		render( <PluginsBrowserList { ...props } /> );
+		const header = screen.queryByText( props.title );
+		expect( header ).toBeVisible();
+		expect( header ).toHaveClass( 'plugins-browser-list__title' );
```

Comment:
Smells like testing an implementation detail, any way this won't be there? Seems like this is testing `classNames`, which properly is redundant. Applicable to a few more instances in this PR.

Reply:
This is more a leftover of the previous tests, I only added those to make sure we're really testing the same element as before. I'm open to removing those.

Reply:
👍 I don't have strong feelings here - I realize they were here before, so leaving it to you to choose whether to keep or remove them.